website/common/script/libs/getItemInfo.js

```diff
@@ -72,7 +72,7 @@ export default function getItemInfo (user, type, item, officialPinnedItems, lang
       itemInfo = {
         key: item.key,
         text: i18n.t('potion', { potionType: item.text(language) }),
-        notes: item.notes(language),
+        notes: item._addlNotes ? item.notes(language) : null,
```

Comment:
is it necessary to check for the existence of `_addlNotes` here since it doesn't seem to be used ? @Alys

Reply:
`_addlNotes` is definitely used for some items so I think the check is needed.

Reply:
Although I haven't looked through the new code in a lot of detail and I might be misunderstanding the question, sorry. :(

Here's an example of where it's used in case that helps:
https://github.com/HabitRPG/habitica/blob/0e590c362318e6e5b8fc525df60ecaa77f64c02d/website/common/script/content/hatching-potions.js#L134-L137

Reply:
Seems like this change is now obsolete since I don't see anymore the error I saw when working on it. So I removed it... Let me know what you think.